### PR TITLE
Upped timeout and added logs for downloading and publishing operation

### PR DESCRIPTION
**Pull request by epananth**
### To double check:

* [ ] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/core-eng/tree/master/Documentation/Validation

### CODE REVIEW

File: src/Microsoft.DotNet.Build.Tasks.Feed/src/PublishArtifactsInManifestBase.cs [hunk 1]
```diff
@@ -427,13 +428,18 @@ public void CheckForStableAssetsInNonIsolatedFeeds()
                             string localSymbolPath = Path.Combine(temporarySymbolsDirectory, symbol);
                             symbolLog.AppendLine($"Downloading symbol : {symbol} to {localSymbolPath}");
 
+                            var downloadStartTime = DateTime.Now;
```
Comment (riarenas) [inline src/Microsoft.DotNet.Build.Tasks.Feed/src/PublishArtifactsInManifestBase.cs:431]: Wouldn't using a stopwatch be better here?

Comment (epananth) [reply src/Microsoft.DotNet.Build.Tasks.Feed/src/PublishArtifactsInManifestBase.cs:431]: Sure will change it

**Review by riarenas — CHANGES_REQUESTED** [review]
I think we should use [Stopwatch](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch?view=net-5.0) instead of DateTime subtraction for our timing

File: src/Microsoft.DotNet.Build.Tasks.Feed/src/PublishArtifactsInManifestBase.cs (hunk 1 above)
Comment (epananth) [reply src/Microsoft.DotNet.Build.Tasks.Feed/src/PublishArtifactsInManifestBase.cs]: changed it to StopWatch

**Review by riarenas — APPROVED** [review]
Do you have any example builds? Just to see how readable the logs are with these changes

**Comment by epananth** [comment]
Running a build now https://dnceng.visualstudio.com/internal/_build/results?buildId=1126019&view=results, I will post the log here after this completes

**Comment by epananth** [comment]
image

**Comment by epananth** [comment]
image
downloading times are not that interesting

**Comment by riarenas** [comment]
I think it would be nice to not lose the decimals in the seconds.

**Comment by epananth** [comment]
Yeah all the downloads were 0 seconds :D , made sure we have decimals for time taken

**Review by riarenas — APPROVED** [review]

**Pull request merged**